js/gmcqView.js

```diff
@@ -0,0 +1,17 @@
+import McqView from 'components/adapt-contrib-mcq/js/mcqView';
+
+class GmcqView extends McqView {
+
+  setupQuestion(...args) {
+    super.setupQuestion(...args);
+  }
```

Comment:
presumably the `setupQuestion` method can be deleted completely as it's not actually doing anything now...